build-system/amp.extern.js

```diff
@@ -35,7 +35,23 @@ window.AMP = {};
 
 // Externed explicitly because we do not export Class shaped names
 // by default.
-window.AMP.BaseElement;
+/**
+ * @constructor
+ */
+window.AMP.BaseElement = function(element) {};
+
+window.AMP.BaseElement.prototype.layout_;
```

Comment:
I see. That's pretty annoying. So, actually, none of these vars are really needed there. We can easily delegate these methods to the element instance as most of methods do now. Not sure why I singled out these properties to be honest :)